gems/aws-sdk-ec2/sig/aws-sdk-ec2/endpoint.rbs

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# WARNING ABOUT GENERATED CODE
+#
+# This file was code generated using smithy-ruby.
+# https://github.com/smithy-lang/smithy-ruby
+#
+# WARNING ABOUT GENERATED CODE
+
+module AWS::SDK::EC2
+  module Endpoint
+    class Params < ::Struct[untyped]
```

Comment:
Minor, but we could think about just EndpointParams and EndpointProvider instead of an endpoint module. Something tells me this is funky as is but I could be wrong.

Reply:
I think it generally fits into how we've structured modules and classes used by the Client.  Eg:  `Builders::OperationName` instead of a top level `OperationNameBuilder` or `Auth::Params` and `Auth::Resolver`.